main/src/ca/uwaterloo/flix/language/phase/Verifier.scala

```diff
@@ -595,6 +595,7 @@ object Verifier {
       case MonoType.Bool    if klazz == classOf[Boolean] => tpe
       case MonoType.Char    if klazz == classOf[Char] => tpe
       case MonoType.Unit    if klazz == classOf[Unit] => tpe
+      case MonoType.Null    if klazz == classOf[Object] => tpe
```

Comment:
This doesn't look correct, because `Null` is also a subtype of e.g. `java.util.List`

`!klazz.isPrimitive` is probably more appropriate

Reply:
Ah yes I overlooked this, you're right thank you.